pathfinder/paizo/campaign_setting/inner_sea_magic/ism_races_companions.lst

```diff
@@ -1,8 +1,8 @@
 SOURCELONG:Inner Sea Magic	SOURCESHORT:ISM	SOURCEWEB:http://paizo.com/products/btpy8mfs?Pathfinder-Campaign-Setting-Inner-Sea-Magic	SOURCEDATE:2011-07
 # Original Entry by: Paul Shipley (elro the onk)
 
-# Race Name		Starting Feats	Size		Move			Face/Space	Reach		Bonus Ability Pool						Combat bonus					Stat bonus					Racial HD Class Skills																																Nb Legs	Nb Hands	Monster Class and Level		Main Race Type	Source Page											Apply Kit
-Eidolon (Fey)	STARTFEATS:1	SIZE:M	MOVE:Walk,20	FACE:5,5	REACH:5	ABILITY:Internal|AUTOMATIC|Racial Traits ~ Eidolon	BONUS:VAR|AC_Natural_Armor|2|TYPE=Base	BONUS:STAT|CON|2	BONUS:STAT|INT|-4	MONCSKILL:Acrobatics|Bluff|Climb|TYPE=Craft|Diplomacy|Disguise|Escape Artist|Fly|Knowledge (Geography)|Knowledge (Nature)|Perception|TYPE=Perform|Sense Motive|Sleight of Hand|Stealth|Swim|Use Magic Device	LEGS:0	HANDS:7	MONSTERCLASS:Eidolon (Fey):1	RACETYPE:Eidolon	SOURCELONG:Advanced Player's Guide	SOURCESHORT:APG	SOURCEPAGE:p.56	KIT:1|Eidolon
+# Race Name	Starting Feats	Size		Move			Face/Space	Reach		Bonus Ability Pool						Combat bonus					Stat bonus					Racial HD Class Skills																																Ability								Nb Legs	Nb Hands	Monster Class and Level		Main Race Type	Source Page											Apply Kit
+Fey Eidolon	STARTFEATS:1	SIZE:M	MOVE:Walk,20	FACE:5	REACH:5	ABILITY:Internal|AUTOMATIC|Racial Traits ~ Eidolon	BONUS:VAR|AC_Natural_Armor|2|TYPE=Base	BONUS:STAT|CON|2	BONUS:STAT|INT|-4	MONCSKILL:Acrobatics|Bluff|Climb|TYPE=Craft|Diplomacy|Disguise|Escape Artist|Fly|Knowledge (Geography)|Knowledge (Nature)|Perception|TYPE=Perform|Sense Motive|Sleight of Hand|Stealth|Swim|Use Magic Device	ABILITY:Special Ability|AUTOMATIC|Low-Light Vision	LEGS:0	HANDS:7	MONSTERCLASS:Eidolon (Fey):1	RACETYPE:Eidolon	SOURCELONG:Advanced Player's Guide	SOURCESHORT:APG	SOURCEPAGE:p.56	KIT:1|Fey Eidolon
```

Comment:
Just noting that this does not follow the standards already set up in the main release. It should be adjusted to use the One Eidolon race set up before being brought into the main release.